### PR TITLE
[128] Paginate freaks

### PR DESCRIPTION
**Pull request by salcudeann**
## Description

- Add cusor-based pagination

## Related
[[42] Paginate Freaks](https://trello.com/c/FSj8oGU7/128-paginate-freaks)

## Steps to Test or Reproduce
Simple, just run the test from _freak_pagination_spec.rb_. Path: _spec/controllers/graphql/queries/freaks_pagination_spec.rb_.

### CODE REVIEW

File: spec/fixtures/responses/queries/Freaks/default.json [hunk 1]
```diff
@@ -0,0 +1,197 @@
+{
+  "status": 200,
+  "body": {
+    "errors": [
```
Comment (alex-damian-negru) [inline spec/fixtures/responses/queries/Freaks/default.json:4]: This is not the expected response.

**Review by alex-damian-negru — CHANGES_REQUESTED** [review]

**Review by alex-damian-negru — APPROVED** [review]
Seems legit :+1:

**Comment by salcudeann** [comment]
[Paginate Freaks](https://trello.com/c/FSj8oGU7/128-paginate-freaks)

**Comment by salcudeann** [comment]
[Paginate Freaks](https://trello.com/c/FSj8oGU7/128-paginate-freaks)

**Pull request merged**